app/javascript/components/patient/common_exposure_cohorts/CommonExposureCohortsTable.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Button, Table } from 'react-bootstrap';
+
+class CommonExposureCohortsTable extends React.Component {
+  render() {
+    return (
+      <Table borderless hover size={this.props.size || 'lg'}>
+        <thead>
+          <tr>
+            <th>Cohort Type</th>
+            <th>Cohort Name/Description</th>
+            <th>Cohort Location</th>
+            {this.props.isEditable && (
+              <React.Fragment>
+                <th></th>
```

Comment:
empty table headers aren't recommended - should be a hidden label on these that screen readers can read